keadm/cmd/keadm/app/cmd/cloud/init.go

```diff
@@ -57,7 +57,7 @@ func NewCloudInit() *cobra.Command {
 		Use:     "init",
 		Short:   "Bootstraps cloud component. Checks and install (if required) the pre-requisites.",
 		Long:    cloudInitLongDescription,
-		Example: cloudInitExample,
+		Example: fmt.Sprintf(cloudInitExample, types.DefaultKubeEdgeVersion),
```

Comment:
this is not correct for now. `types.DefaultKubeEdgeVersion` is 1.12.0 but without prefix `v`
Now we distiguish `v1.12.0`(with prefix `v`) and `1.12.0` (without prefix `v`) in installation, some errors happen when we mix them up.
we should optimize it, so that installation is always successful during installation both with and without prefix.
@my-git9

Reply:
you can try it if you have interest :)

Reply:
The test is ok:
<img width="899" alt="image" src="https://user-images.githubusercontent.com/76980726/199154817-47098088-546c-4760-bc3d-4e06eb9ec6c3.png">

Reply:
yes, it is.
I mean there're some other codes using kubeedge-version like keadm join/deprecated init/deprecated join etc. these code can be optimized.

Reply:
OK, I can try it later, thanks